index.html

```diff
@@ -76,6 +129,22 @@ <h2 id=intro>Introduction</h2>
 by the internationalization Working Group
 for more information.)
 
+    <aside class=note title="Ruby as notes">
+      Notes are sometimes placed between lines
+      similarly to how ruby is laid out
+      (inter-linear notes).
+      <span class="checkme">
+      The processing of this arrangement is not covered in this document.
+      Putting line break within ruby string is not allowed for mono-ruby. 
+      In addition, putting line break within ruby base text is not allowed 
+      for group-ruby. 
```

Comment:
It would be useful to link to the definition of mono-ruby and group-ruby (since these terms haven't been introduced yet if the reader reads from top to bottom and isn't quite familiar with Japanese typography).

Reply:
Let me find suitable point in JLReq and insert links.

Reply:
to be done as a part of #18 (https://github.com/w3c/simple-ruby/issues/19#issuecomment-627149240)